templates/default.nginx_http.conf.template

```diff
@@ -99,3 +99,8 @@ server {
         root   /usr/share/nginx/html2;
     }
 }
+
+server {
+    listen 4200 default_server;
+    return 301 https://{{ DOMAIN_NAME }};
```

Comment:
Do we need to tack `$request_uri` onto the end of the return URL?  Possibly, without it, this would redirect all unmatched requests to the root URL.

Reply:
Eh, might as well. 
I suppose it couldn't hurt

Reply:
Cool, this way, the request
  protocol://4.5.6.7/some/path
will map to
  https://{{DOMAIN_NAME}}/some/path
rather than
  https://{{DOMAIN_NAME}}
and the search bot will be able to index.